sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/schema.yaml

```diff
@@ -0,0 +1,112 @@
+fields:
+- name: customer_id
+  type: STRING
+  mode: NULLABLE
+- name: subscription_id
+  type: STRING
+  mode: NULLABLE
+- name: original_subscription_id
+  type: STRING
+  mode: NULLABLE
+- name: plan_id
+  type: STRING
+  mode: NULLABLE
+- name: status
+  type: STRING
+  mode: NULLABLE
+- name: event_timestamp
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: customer_start_date
+  type: TIMESTAMP
+  mode: NULLABLE
```

Comment:
For the future reference, how do you chose the order of the fields?

Reply:
It just depends on the order of columns in the query.  Using `SELECT *` means you'll get the same column order as the underlying table/CTE, and if you want to have things in a particular order you can instead specify the columns to select instead of `*` (e.g. in the final `SELECT` of the query).

Reply:
Got it! Thank you!!!

Reply:
The fields are reordered.